doc/CCPPtechnical/source/ScientificDocRules.inc

```diff
@@ -420,8 +412,8 @@ Doxygen Configuration
 Configuration File
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-The CCPP contains a doxygen configuration file
-``./ccpp/physics/physics/docs/ccpplatex_dox``, such that you don’t need to
+The CCPP is distributed with a doxygen configuration file
+``./ccpp/physics/physics/docs/ccpp_doxyfile``, such that you don’t need to
```

Comment:
I have created a new file called ccpp_doxyfile because we should not have the  current code use a configuration file associated with the V3 release. I will create a PR to ccpp-physics/docs to include this new file in gmtb/develop.